### ISSUE TITLE
Loginpage Accessibility

### ISSUE THREAD

**Issue by JBartscher** [cloudogu/cas#77]
For the Loginpage of the CAS, which is a JSP Site, it would be nice if accessibility-tools like screenreaders get information regarding the status of a login attempt (eg. read out the alert in case of a failed login).
Also all functional elements should be accessible via the keyboard.
